packages/paste-website/src/components/site-wrapper/sidebar/SidebarNavigation.tsx

```diff
@@ -1,19 +1,19 @@
 import * as React from 'react';
-import {graphql, useStaticQuery} from 'gatsby';
-import {Box} from '@twilio-paste/box';
+import { graphql, useStaticQuery } from 'gatsby';
+import { Box } from '@twilio-paste/box';
 import {
   useDisclosurePrimitiveState,
   DisclosurePrimitive,
   DisclosurePrimitiveContent,
 } from '@twilio-paste/disclosure-primitive';
-import {SidebarAnchor} from './SidebarAnchor';
-import {SidebarDisclosureButton} from './SidebarDisclosureButton';
-import {SidebarItem} from './SidebarItem';
-import {SidebarNestedItem} from './SidebarNestedItem';
-import {SidebarNestedList} from './SidebarNestedList';
-import {PackageStatus, SidebarCategoryRoutes} from '../../../constants';
-import {getCurrentPathname, getNameFromPackageName, getHumanizedNameFromPackageName} from '../../../utils/RouteUtils';
-import {filteredComponents} from '../../../utils/componentFilters';
+import { SidebarAnchor } from './SidebarAnchor';
+import { SidebarDisclosureButton } from './SidebarDisclosureButton';
+import { SidebarItem } from './SidebarItem';
+import { SidebarNestedItem } from './SidebarNestedItem';
+import { SidebarNestedList } from './SidebarNestedList';
+import { PackageStatus, SidebarCategoryRoutes } from '../../../constants';
```

Comment:
These spaces make the prettier job on CI fail. Not sure how they got added but you'll need to run this file through prettier for it to pass and so we can merge. LMK if you need assistance on this.